docs/getting-started/user-guides/paymetheus.md

```diff
@@ -0,0 +1,60 @@
+# **Paymetheus** #
+
+Paymetheus is a simple to use graphical wallet for Decred. With it, you can send and receive DCR, 
+purchase tickets for [PoS voting](/mining/proof-of-stake.md), get a history of all your transactions and more.
+About the only thing Paymetheus doesn't do is PoS voting.
+
+---
+
+## **Download and Install** ##
+The first thing you'll need to do is get a copy of Paymetheus. You will find the downloads [here](https://github.com/decred/decred-binaries/releases). 
+Download the file for your operating system and install. Once you're done, run the 'Decred' program.
+You'll notice a program called "Paymetheus Standalone". This does not include the 'dcrd' program 
+required to connect to the Decred network, so don't use this one.
```

Comment:
I write these guides assuming that people have zero understanding of cryptography, networking or even command line usage. They just want to download a program and start using Decred and don't want to know how it works under the hood. As such, I prefer to give people a single way of doing things in these basic guides. By saying "don't use this" it removes possible confusion. 

There are reasons to use the standalone such as running the dcrd server remotely, but this is an advanced topic (remembering the target audience) and so I have purposely omitted discussing this. There are other places this information can go, and can be linked from here, but it's outside the scope of a beginners guide.